preorder/package.json

```diff
@@ -25,5 +25,6 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
-  }
+  },
+  "proxy": "http://localhost:8080"
```

Comment:
?

Reply:
This is for dev only.  If the webpack dev server can't find the requested resource, it falls back to the proxy setting.  This allows you to run the API server locally without getting CORS errors.  This is how I always setup my projects when there is an API server.  If you had a different setup during development, I'd love to hear about it. 

Note that if you "namespace" your API like /api/adduser /api/addorder, then you can only proxy requests under /api, but that usually doesn't make any difference.  You can read more about this setting here: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#proxying-api-requests-in-development